README.md

```diff
@@ -500,16 +555,17 @@ NOTE
 
 [Back to content](#content)
 
-##See also
+## See also
 ----------
 * [Tarantool](http://tarantool.org) homepage.
```

Comment:
BTW, should we update the link here to tarantool.io (and with https)?

Reply:
Resolved in #25